src/components/SideTray/Body/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import style from './style.css';
+
+const Body = ({ children, ...passedProps }) => {
+  return (
+    <div className={style.SideTray__Body} {...passedProps}>
```

Comment:
Style name could just be `Body`, since css modules should be taking care of uniqueness.

And you should probably concat `style.Body` to whatever `props.className` the consumer passed in.

Reply:
If you take a look at the `Media.jsx` in tk-ui I have a pattern that does this exact thing if that helps